react/package.json

```diff
@@ -29,7 +29,7 @@
     "react-router": "^5.3.4",
     "react-router-dom": "^5.3.4",
     "react-scripts": "^5.0.1",
-    "typescript": "^4.9.3",
+    "typescript": "^4.8.2",
```

Comment:
Do we want to use an older version of TS?

Reply:
I'm not sure why this change is here, will remove.